ui/src/components/group/GroupReviewTable.js

```diff
@@ -115,52 +124,66 @@ class GroupReviewTable extends React.Component {
                 });
                 return;
             }
+            // show prompt for user to ask for confirmation once the user asked to delete member/s
+            let showDeleteConfirmation =
+                CollectionUtils.intersection(
```

Comment:
why is taking intersection required here? Wouldn't checking the size of set `deletedMembers` suffice for this condition?